libraries/Microsoft.Bot.Builder.AI.TriggerTrees/RelationshipType.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Microsoft.Bot.Builder.AI.TriggerTrees
+{
+    /// <summary>
+    /// Relationship between trigger expressions.
+    /// </summary>
+    public enum RelationshipType
```

Comment:
You're probably just moving this code, so just a thought and no need tochange right now: I'd rather something more self-explanatory like ExpressionRelationships or TriggerRelationships